poll.go

```diff
@@ -148,6 +148,9 @@ func pollReqID(ctx context.Context, reqID, url string) (bool, error) {
 func pollREST(ctx context.Context, reqID string) (*msg.QueryResult, string, error) {
 	var resp *http.Response
 	result, err := restV1Client.Result(ctx, reqID, zapi.Response(&resp))
+	if err != nil {
+		return nil, "", errors.Wrap(err, "couldn't get result from client")
+	}
```

Comment:
We still want to get the traceID if at all possible, even if there is an error.

I was thinking we could just do:

```go
var traceID string
if resp != nil {
    traceID = resp.Header.Get("uber-trace-id")
}
```

not sure if we also need to check if `resp.Header != nil`